src/styles/blocks/product-list.scss

```diff
@@ -0,0 +1,13 @@
+.product-list {
+  box-sizing: content-box;
+  padding: 50px 40px 0;
+  margin: 0 auto;
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: space-between;
+  width: 944px;
```

Comment:
Checklist #13

Reply:
Okey. Hardcoded it through a function :)